hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/ratis/CSMMetrics.java

```diff
@@ -191,14 +191,14 @@ public long getNumBytesCommittedCount() {
     return numBytesCommittedCount.value();
   }
 
-  public MutableRate getApplyTransactionLatency() {
-    return applyTransaction;
+  public MutableRate getApplyTransactionNsLatency() {
```

Comment:
nit: 
```suggestion
  public MutableRate getApplyTransactionLatencyNs() {
```

Ms and Ns are used as suffix in variable names but in functions it is not consistent. Can you please change it here as well as  `recordApplyTransactionNsCompletion` to `recordApplyTransactionCompletionNs` and `recordWriteStateMachineNsCompletion` to`recordWriteStateMachineCompletionNs`? Similar to `incPipelineLatencyMs`.